README.md

```diff
@@ -2,7 +2,7 @@
 
 # Ledger Boilerplate Application
 
-This is a boilerplate application which can be forked to start a new project for the Ledger Nano S/X/SP and Stax.
+This is a boilerplate application which can be forked to start a new project for the Ledger Nano X/SP, Stax and Flex.
 
```

Comment:
Add a comment about the dedicated nanos branch

Reply:
nanos branch is only used for internal usage